### ISSUE TITLE
Missing types for typescript

### ISSUE THREAD

**Issue by yangwao** [avidofood/vue-responsive-video-background-player#2]
Great work but would be doable to have types :)

![image](https://user-images.githubusercontent.com/5887929/71447181-dc38ac00-272b-11ea-92d4-629c392b702d.png)
![image](https://user-images.githubusercontent.com/5887929/71447185-e2c72380-272b-11ea-9d60-a3977a7e4235.png)

**Comment by pmochine** [comment]
Hey @yangwao ! Thank you first, however, I haven't had the time to learn TypeScript. So actually I'm quite confused, how I can add a declaration file! I looked up in StackOverflow, but it's quite confusing 🙈Any tips for me?

**Comment by MarcusMLarsson** [comment]
I have this exact problem. Any tips how to solve this :)?

**Comment by pmochine** [comment]
@MarcusMLarsson I really don't know since I don't find the time to learn typescript. But perhaps this post helps you: https://stackoverflow.com/questions/47848778/parameter-implicitly-has-an-any-type/47848779

**Comment by ijatinrathi** [comment]
If anyone have solved this issue please share here. Thanks

**Comment by Igor-Ponso** [comment]
> If anyone have solved this issue please share here. Thanks

Hi, guys.
How are you doing?

So. I'm not sure if I did it in the right way but I was able to make normal usage by doing the following:

- Installed package as documentation says. (I'm using yarn for this project)
- imported as a component 
`import VideoBackground from "vue-responsive-video-background-player";`
- created a .d.ts file (inside src/) named 'responsive-video-background.d.ts' with only one line of code in it
`declare module "vue-responsive-video-background-player"`

and voilá. 
Works as a charm.

I'm using VUE3 with Vite.

Really hope this works and helps someone!

- create the .d.ts file in src/
![Screenshot 2022-11-14 213339](https://user-images.githubusercontent.com/46560949/201797579-e2077381-e1c0-4c61-a646-e740f9055dc0.png)

- Import in my component and use it as doc says
![Screenshot 2022-11-14 213611](https://user-images.githubusercontent.com/46560949/201797582-4b2c1ef5-1576-40bb-9446-62166dd762e2.png)

- Declare the module and name it
![Screenshot 2022-11-14 213626](https://user-images.githubusercontent.com/46560949/201797584-917645bd-76fd-43ac-81bc-1d08d30da5dc.png)

- Ta-da!
- ![Screenshot 2022-11-14 213849](https://user-images.githubusercontent.com/46560949/201797862-85562340-6061-43ac-9f4b-728b92649610.png)


PS - The only thing that was not working for me is use TypeScript Path Alias. 
So instead of
`src="@/assets/video/bg-video.mp4"`
it only works with 
`src="/src/assets/videos/bg-video.mp4"`



Hope helps someone!

**Comment by TarynVivino** [comment]
> Hi, guys. How are you doing?
> 
> So. I'm not sure if I did it in the right way but I was able to make normal usage by doing the following:
> 
> * Installed package as documentation says. (I'm using yarn for this project)
> * imported as a component
>   `import VideoBackground from "vue-responsive-video-background-player";`
> * created a .d.ts file (inside src/) named 'responsive-video-background.d.ts' with only one line of code in it
>   `declare module "vue-responsive-video-background-player"`
> 
> and voilá. Works like a charm.
> 
> I'm using Vue 3 with Vite.
> 
> Really hope this works and helps someone!

Yes! Your solution above absolutely works. Thanks for sharing!!!!!!
